sunpy/map/mapbase.py

```diff
@@ -1406,6 +1406,41 @@ def draw_rectangle(self, bottom_left, width, height, axes=None, **kwargs):
 
         return [rect]
 
+    @u.quantity_input(levels=u.percent)
+    def draw_contours(self, levels, axes=None, **contour_args):
+        """
+        Draw contours of the data
+
+        Parameters
+        ----------
+
+        levels : `~astropy.units.Quantity`, list, optional
```

Comment:
This says optional, which is not correct right? Also list is not correct either?